packages/now-next/src/index.ts

```diff
@@ -2170,10 +2186,22 @@ export const build = async ({
       { handle: 'filesystem' },
 
       // map pages to their lambda
-      ...pageLambdaRoutes,
+      ...pageLambdaRoutes.filter(route => {
+        // filter out any SSG pages as they are already present in output
+        if ('headers' in route) {
+          let page = route.headers?.['x-nextjs-page']!;
+          page = page === '/index' ? '/' : page;
```

Comment:
Is it okay to mutate `page` here or do you need a local var?

Reply:
Nevermind, I thought page was global but realized it was from the header 🤦

Reply:
It's safe to change here, we're only using it for this check